sprintsLogs/SprintsLogs.log

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
Please add this exclusion to .gitignore in order to avoid uploading unnecessary files or folders.
/sprintsLogs